wordpress-mobile/WordPress-Android
# Update Firebase Test Lab credentials

The Firebase credentials currently in use have been revoked. This updates them. It will allow the nightly connected tests to run again.

To test:

Its difficult to test without the scheduled job. You can see the credentials work in the other related PRs: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1280, https://github.com/wordpress-mobile/AztecEditor-Android/pull/822

Update release notes:
